recipes/bftools/meta.yaml

```diff
@@ -0,0 +1,27 @@
+{% set version = "5.7.1" %}
+{% set sha256 = "ed5e378f210a43eec90fc5d51710489377af47c981adc3b0785a4da5d2e13b18" %}
+
+
+package:
+  name: bftools
+  version: '{{version}}'
+
+source:
+  url: "http://downloads.openmicroscopy.org/latest/bio-formats{{version}}/artifacts/bftools.zip"
+  sha256: '{{sha256}}'
+build:
+  number: 0
+  noarch: generic
+
+requirements:
+  run:
+    - openjdk >=8
+
+test:
+  commands:
+    - bfconvert
```

Comment:
Looks good, but the tests are failing ... maybe because this tool is again writing to stderr?

Reply:
Return code is wrong. I try printing the version instead of help.